browser_windows.go

```diff
@@ -1,16 +1,13 @@
+//go:generate mkwinsyscall -output zbrowser_windows.go browser_windows.go
+//sys ShellExecute(hwnd int, verb string, file string, args string, cwd string, showCmd int) (err error) = shell32.ShellExecuteW
 package browser
 
-import (
-	"os/exec"
-	"strings"
-	"syscall"
-)
+import "os/exec"
+const SW_SHOWNORMAL = 1
```

Comment:
please move this to line 8-9, this constant so the constant isn't exported by the package.